### PR TITLE
Anton card layout

### PR DESCRIPTION
**Pull request by AntonBaranskyi**
Finiish with the phone card (used draft photos and logic)

image

### CODE REVIEW

File: src/assets/icons/Heart_filled.svg
Comment (maxyma-sanx) [inline src/assets/icons/Heart_filled.svg:1]: Unnecessary file

File: src/assets/icons/heart.svg
Comment (maxyma-sanx) [inline src/assets/icons/heart.svg:1]: Unnecessary file

File: src/assets/icons/hearts.svg
Comment (maxyma-sanx) [inline src/assets/icons/hearts.svg:1]: Unnecessary file

File: src/assets/phone_img.png
Comment (maxyma-sanx) [inline src/assets/phone_img.png:1]: Unnecessary file

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+
+import heart from '../../../assets/icons/hearts.svg';
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: imports now from sprite

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  padding: 32px;
+  width: 272px;
+
+  border-radius: 8px;
+  border: ${(props) => `1px solid ${props.theme.colors.grayElements}`};
+  background: #fff;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: color from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+  margin: 0 auto;
+
+  margin-bottom: 24px;
+`;
+
+export const CardTitle = styled.h2`
+  color: ${({ theme }) => theme.colors.grayPrimary};
+
+  font-size: 14px;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: fonts from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  gap: 8px;
+
+  margin-bottom: 8px;
+  border-bottom: 1px solid #e2e6e9;
+
+  padding-bottom: 8px;
+`;
+
+export const CurrentPrice = styled.h3`
+  font-size: 22px;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: fonts from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+  margin-bottom: 8px;
+`;
+
+export const PriceWrapper = styled.div`
+  display: flex;
+  gap: 8px;
+
+  margin-bottom: 8px;
+  border-bottom: 1px solid #e2e6e9;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: colors from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+  padding-bottom: 8px;
+`;
+
+export const CurrentPrice = styled.h3`
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 140%;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: line height from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+  color: #0f0f11;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: colors from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`;
+
+export const DescrTitle = styled.h3`
+  color: ${({ theme: { colors } }) => colors.graySecondary};
+
+  font-size: ${({ theme }) => theme.fonts.sizeXxs};
+  font-weight: 600;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: fonts from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+  line-height: normal;
+`;
+
+export const DescrValue = styled.p`
+  color: ${({ theme }) => theme.colors.grayPrimary};
+  text-align: right;
+  font-family: Mont;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx:89]: this font we have by default

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+  display: flex;
+  justify-content: space-between;
+  gap: 8px;
+`;
+
+export const ButtonAdd = styled.button<ButtonProps>`
+  width: 100%;
+  height: 40px;
+
+  color: ${(props) => (props.isClicked ? '#216cff' : '#fff')};
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: colors from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+
+  border: ${(props) => (props.isClicked ? '1px solid #e2e6e9' : 'none')};
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: colors from themes

File: src/module/shared/PhoneCard/PhoneCard.styled.tsx [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+  cursor: pointer;
+
+  border-radius: 8px;
+  background: ${(props) =>
+    props.isClicked
+      ? props.theme.colors.white
+      : props.theme.colors.accentPrimary};
+
+  font-size: ${({ theme }) => theme.fonts.sizeXs};
+  font-style: normal;
```
Comment (maxyma-sanx) [inline src/module/shared/PhoneCard/PhoneCard.styled.tsx]: fonts from themes

**Review by maxyma-sanx — CHANGES_REQUESTED** [review]

**Review by KandaskalovOleg — APPROVED** [review]

**Review by maxyma-sanx — APPROVED** [review]

**Pull request merged**